packages/medusa/src/repositories/publishable-api-key-sales-channel.ts

```diff
@@ -16,35 +19,38 @@ export const PublishableApiKeySalesChannelRepository = dataSource
       publishableApiKeyId: string,
       config?: { q?: string }
     ): Promise<SalesChannel[]> {
-      const query = this.createQueryBuilder("PublishableKeySalesChannel")
-        .select("PublishableKeySalesChannel.sales_channel_id")
-        .innerJoinAndMapOne(
-          "PublishableKeySalesChannel.sales_channel_id",
-          SalesChannel,
-          "SalesChannel",
-          "PublishableKeySalesChannel.sales_channel_id = SalesChannel.id"
-        )
+      const salesChannelAlias = "sales_channel"
+
+      const queryBuilder = this.createQueryBuilder(
```

Comment:
@fPolic here are the changes I was talking about, if you could look at it and let me know what ou think

Reply:
LGTM @adrien2p :)